app/middleware/postcodeValidator.js

```diff
@@ -0,0 +1,48 @@
+const log = require('../lib/logger');
+const postcodeValidator = require('../lib/postcodeValidator');
+const isNotEnglishPostcode = require('../lib/isNotEnglishPostcode');
+const PostcodesIO = require('postcodesio-client');
+
+const postcodes = new PostcodesIO();
```

Comment:
Maybe const PostcodesIO = require('postcodesio-client')(); (with postcodes renamed) would be better.